docs/content/configuration/configuring_hostdefiner.md

```diff
@@ -7,7 +7,7 @@ For more information about using the host definer, see [Using dynamic host defin
 |Field|Description|
 |---------|--------|
 |`prefix`|Adds a prefix to the hosts defined by the host definer.<br>**Note:** The prefix length is bound by the limitation of the storage system. When defined, the length is a combination of both the prefix and node (server) hostname.|
-|`connectivityType`|Selects the connectivity type for the host ports.<br>Possible input values are:<br>- `nvmeofc` for use with NVMe over Fibre Channel connectivity<br>- `fc` for use with Fibre Channel over SCSI connectivity<br>- `iscsi` for use with iSCSI connectivity<br>By default, this field is blank and the host definer selects the first of available connectivity types on the node, according to the following hierarchy: NVMe, FC, iSCSI.|
+|`connectivityType`|Selects the connectivity type for the host ports.<br>Possible input values are:<br>- `nvmeofc` for use with NVMe over Fibre Channel connectivity<br>- `fc` for use with Fibre Channel over SCSI connectivity<br>- `iscsi` for use with iSCSI connectivity<br>By default, this field is blank and the host definer selects the first of available connectivity types on the node, according to the following hierarchy: NVMe, FC, iSCSI.<br>**Note:** When left blank, the connectivity type will update along with any changes within the host ports, according to the set hierarchy. If the value is set and there are host port changes, connectivity needs to be manually updated. For more information, see [Changing node connectivity](../using/changing_node_connectivity.md).|
```

Comment:
It's documented here, as it was last release.

`By default, this field is blank and the host definer selects the first of available connectivity types on the node, according to the following hierarchy: NVMe, FC, iSCSI.`

Reply:
I think having a main place with the hierarchy, and then the hierarchy will be in a link and it can be easier, but if you think it's good enough for the user so I am ok with it

Reply:
this placement is the best placement for it. i also mention it in other places. i think we can resolve this thread.

Reply:
ok